opensfm/data/sensor_data.json

```
@@ -3641,6 +3641,7 @@
     "Yakumo Mega Image X": 7.11,
     "Yakumo Mega Image XL": 4.23,
     "Yakumo Mega Image XS": 6.4,
+    "YTXJ02FM": 6.17,
```

```suggestion
    "unknown YTXJ02FM": 6.17,
```
It is not elegant but it works and also complies with the `"make model"` scheme.